vgaidarji/github-actions-pr-merger
# Dont trigger dry run on PRs automatically

### What has been done
1. Eradicated `dry-run` action configuration. Triggering action in `dry-run` mode for PR builds makes not much sense since main action purpose is to be triggered via PR comments. Having no `comment` object present in GitHub payload in action workflow build makes it impossible to perform full integration test on each PR. Instead, to test the action comment with `dry-run` parameter should be left on PR.

See https://github.com/vgaidarji/github-actions-pr-merger/pull/20 for more context.

### How to test
No easy way to test atm.
When `/robin merge --dry-run` command support is added, trigger a build using via that command which will execute `.github/test-action-on-pr-comment.yml` workflow for comment left on PR. 

1. Merge PR to master so that `.github/test-action-on-pr-comment.yml` workflow changes take effect.
2. Open another PR for any future changes and leave a comment on PR with `/robin merge --dry-run` 
3. Verify action `test-action-on-pr-comment.yml` workflow is triggered and dry run merge operation result is posted to PR
